### PR TITLE
Including multiple datacenter and cluster tests [skip ci]

### PR DESCRIPTION
**Pull request by rajanashok**
Added the multiple datacenter and cluster tests to nightly run.
Disabling certs for an unblocked nightly run.

### CODE REVIEW

File: tests/nightly/nightly-kickoff.sh [hunk 1]
```diff
@@ -123,7 +164,7 @@ if [ $buildStatus -eq 0 ]
 then
 echo "Success"
 cat <<EOT >> nightly_mail.html
-To: rashok@vmware.com
+To: mwilliamson-staff-adl@vmware.com
```
Comment (mhagen-vmware) [inline tests/nightly/nightly-kickoff.sh]: not the whole staff right now, just matt you and me for now

**Comment by mhagen-vmware** [comment]
lgtm

**Comment by mdubya66** [comment]
lgtm

**Comment by rajanashok** [comment]
merging

**Comment by rajanashok** [comment]
merging

**Pull request merged**